config/jobs/cert-manager-csi/cert-manager-csi-presubmits.yaml

```diff
@@ -66,7 +66,7 @@ presubmits:
         - e2e
         resources:
           requests:
-            cpu: 6
+            cpu: 3500m
             memory: 12Gi
```

Comment:
Do the memory requests add up/are they proportional to the instance size and CPUs too?